test/language/expressions/class/fields-computed-name-static-propname-constructor.js

```diff
@@ -0,0 +1,32 @@
+// This file was procedurally generated from the following sources:
+// - src/class-fields/static-propname-constructor.case
+// - src/class-fields/propname-error/cls-expr-computed-name.template
+/*---
+description: Static class field cannot have PropName 'constructor' (computed)
+esid: sec-class-definitions-static-semantics-early-errors
+features: [class-fields]
+flags: [generated]
+info: |
+    Static Semantics: PropName
+    ...
+    ComputedPropertyName : [ AssignmentExpression ]
+      Return empty.
+
+    
+    // This test file tests the following early error:
+    Static Semantics: Early Errors
+
+      ClassElement : staticFieldDefinition;
+        It is a Syntax Error if PropName of FieldDefinition is "prototype" or "constructor".
+
+---*/
+
+
+var x = "constructor";
```

Comment:
`var constructor = "/* { name } */"`

using `constructor` for the variable name should fun (in the template)

The template may also have a better description why computed name does not produce an early error.